test/built-ins/Temporal/Calendar/prototype/prop-desc.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2020 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal-calendar-objects
```

Comment:
```suggestion
esid: sec-temporal-calendar-prototype
```

Same for the other files

Reply:
👍 Done.